std/haxe/web/Scheme.hx

```diff
@@ -0,0 +1,57 @@
+package haxe.web;
+/**
+* The scheme consists of a sequence of characters beginning with a letter and followed by any combination of letters, digits, plus (+), period (.), or hyphen (-). Although schemes are case-insensitive, the canonical form is lowercase and documents that specify schemes must do so with lowercase letters. It is followed by a colon (:).
+* @class Scheme
+* @module Tamina
+*/
+@:enum abstract Scheme(String) from String to String  {
```

Comment:
> Former is a valid reason, latter is an invalid reason.

Says who?

Also, the difference you're making is pure fiction. It's not like an `FsOpenFlag` will ever "come from an external source". And even if it did, the correct way would be properly parse the string, rather to just blindly accept it and run into an error down the road.